src/vs/base/browser/markdownRenderer.ts

```diff
@@ -108,13 +108,13 @@ export function renderMarkdown(markdown: IMarkdownString, options: MarkdownRende
 		let attributes: string[] = [];
 		if (href) {
 			({ href, dimensions } = parseHrefAndDimensions(href));
-			attributes.push(`src="${href}"`);
+			attributes.push(`src="${href.replace(/"/g, '&quot;')}"`);
```

Comment:
This should likely be extracted to its own function instead of repeated

Reply:
Done